service/csv_service.go

```diff
@@ -0,0 +1,53 @@
+package service
+
+import (
+	"encoding/csv"
+	"io"
+	"os"
+	"WizelineApi/domain/model"
+	//"fmt"
+	"errors"
+	"strconv"
+)
+
+//Funcion para leer un CSV. 
+func ReadCSV(path string, pkms *[]model.Pokemon) (error){
+	
+	file, err := os.Open(path)
+	
+	if err != nil {
+		return  errors.New("Couldn't Open  File")
+	}
+
+	r:= csv.NewReader(file)
+
+	//Loop para leer todos los records del CSV
+	for {
+		//Lee el primer (row) del csv. 
+		record, errCsv := r.Read()
+
+		//io.EOF es el error generado cuando se llega al final del archivo
+		if errCsv == io.EOF{ //cuando llegue al final del archivo, se realiza el break del loop for.
+			break
+		}
+
+		if errCsv != nil {
+			return errCsv 
```

Comment:
You should wrap your errors when these come from an external package. 

This will be helpful when trying to debug the root cause of a bug.